rocketpy/monte_carlo/mc_environment.py

```diff
@@ -0,0 +1,51 @@
+__author__ = "Mateus Stano Junqueira"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+from typing import Any, List, Tuple, Union
+
+from pydantic import Field, StrictInt, StrictStr, validator
+
+from ..Environment import Environment
+from .DispersionModel import DispersionModel
+
+# TODO: name suggestions: `DispersionEnvironment`, `DispEnvironment`, EnvironmentDispersion`, EnvironmentDisp`, `MonteCarloEnvironment`, `EnvironmentMonteCarlo`,
```

Comment:
Tks for all the suggestions. I believe the most intuitive name would be `MonteCarloEnvironment` here. @MateusStano agreed? If not, we may need a third view to help us.

Reply:
Thinking back on this I really liked the `Data` names: `EnvironmentData`. I can understand if it is not descriptive enough though.

I would say that I like `DispersionEnvironment` just as much as `MonteCarloEnvironment` heheh

I think we gonna need to open a poll with the team to decide this one

Reply:
Personally not a fun of polling but I agree we need it. Let's finish it by this week please.